Formula/chunkwm.rb

```diff
@@ -0,0 +1,108 @@
+class Chunkwm < Formula
+  desc "Tiling window manager for macOS based on plugin architecture"
+  homepage "https://github.com/koekeishiya/chunkwm"
+  url "https://github.com/koekeishiya/chunkwm/archive/v0.2.28.tar.gz"
+  sha256 "917c6aa09521ff4b7463f731192eea6be6313f0fa4e585f63f85603d0993cf92"
+
+  head do
+    url "https://github.com/koekeishiya/chunkwm.git"
+  end
+
+  option "without-tiling", "Do not build tiling plugin."
+  option "without-ffm", "Do not build focus-follow-mouse plugin."
+  option "without-border", "Do not build border plugin."
+  option "with-transparency", "Build transparency plugin."
+
+  def install
+    # create plugins and example directories
+    (share/"plugins").mkpath
+    (share/"examples").mkpath
+
+    # install chunkwm
+    system "make", "install"
+    inreplace "#{buildpath}/examples/chunkwmrc", "~/.chunkwm_plugins", "$(brew --prefix chunkwm)/share/plugins"
```

Comment:
Yeah, I noticed the configuration file, but presumably you're just teaching it a new path rather than actually giving it a command to execute routinely? In which case`#{pkgshare}` will be automatically expanded by Homebrew to the full path.